validator/validator.protoascii

```diff
@@ -2278,6 +2278,23 @@ tags: {
     supported_layouts: NODISPLAY
   }
 }
+# <amp-embed>
```

Comment:
Please add a comment here and with `amp-ad` that changes should be reflected in the other respectively.